### ISSUE TITLE
Can you update Python to 3.5.0?

### ISSUE THREAD

**Issue by Jeewsus** [saltstack/salt-winrepo#616]

**Comment by TheBigBear** [comment]
@Jeewsus any particular reasons to do so? 

Anything that doesn't work for you right now? 

AFAIK the ```'core'``` salt code will be going to python 3.x at some point in the future, but what are your reasons for asking for it? 

And why are you asking for it on the old ```'salt-winrepo'``` part of the project? 

The ```'salt-winrepo'``` is old and has been superceeded by ```'salt-winrepo-ng'``` for all current salt clients >= 2015.8.x

**Comment by Jeewsus** [comment]
@TheBigBear Yeah, i would like to update the computers i manage all to 3.5 since some people would like to use it. 
And im on 2015.5.x with my minions actually.
